crmsh/bootstrap.py

```diff
@@ -563,6 +567,10 @@ def join_sbd(self, peer_host):
         dev_list = self._get_sbd_device_from_config()
         if dev_list:
             self._verify_sbd_device(dev_list, [peer_host])
+        else:
+            vote_dict = utils.get_quorum_votes_dict(peer_host)
+            if int(vote_dict['Total']) < 2:
+                warn(self.DISKLESS_SBD_WARNING)
```

Comment:
On join side, when finding that init node's actual votes less than 2, give warning

Reply:
It seems to make more sense to check the "Expected votes", which means whether the "configuration"/"setup" is going to meet the requirement.

Reply:
Changed, thanks!